testing/integration/endpoint_test_tools.go

```diff
@@ -128,6 +128,6 @@ func installElasticDefendPackage(t *testing.T, info *define.Info, policyID strin
 		t.Logf("Error installing fleet package: %v", err)
 		return r, fmt.Errorf("error installing fleet package: %w", err)
 	}
-	t.Logf("Endpoint package Policy Response:\n%+v", pkgResp)
```

Comment:
was this log not useful for debugging when tests fail ?

Reply:
the response is returned regardless, even on failure. So it's better to log the response if and only if an error happens. Here it was being logged on success as well.